flink-libraries/flink-table/src/test/scala/org/apache/flink/table/runtime/datastream/DataStreamUserDefinedFunctionITCase.scala

```diff
@@ -230,6 +231,26 @@ class DataStreamUserDefinedFunctionITCase extends StreamingMultipleProgramsTestB
     assertEquals(expected.sorted, StreamITCase.testResults.sorted)
   }
 
+
+  @Test
+  def testScalarFunctionDeterministic(): Unit = {
```

Comment:
@Xpray  @sunjincheng121 I mean we can remove this test, and add a test in `ExpressionReductionTest`.